acceptable/__main__.py

```diff
@@ -33,6 +33,7 @@ def main():
         cli_args = parse_args()
         sys.exit(cli_args.func(cli_args))
     except Exception as e:
+        raise
```

Comment:
Will this not prevent the sys.exit from running?

Reply:
Gah, that was debugging code left in, will remove, thanks